packages/ui/src/components/accordion/accordion.native.tsx

```diff
@@ -4,8 +4,14 @@ import { useTheme } from '@shopify/restyle';
 
 import { ChevronDownIcon, ChevronUpIcon, Text, Theme, TouchableOpacity } from '../../../native';
 
+interface AccordionProps {
+  content: ReactNode;
+  label: string;
+  testID?: string;
```

Comment:
Typ with a prop, we would use `testId`.

Reply:
Yeah I looked into this because it seems so obviously ugly and inconsistent to use `testID` to my eye. But... it's in the Maestro docs as `testID` 🤷🏼 

`ID` capital `D` suggests it's the _Identity Document_ acronym, rather than just the word, `Identifier`.

I would much rather remain consistent with web frameworks and the rest of our test stacks that quite consistently uses `testid. If possible, I think we should. But if this is the RN way of doing it, then I have to accept it with gritted teeth.

Reply:
Ok, thanks for feedback here. I'll approve PR, lgtm otherwise.